docs/tutorials/chains/Arbitrum.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: i18n_page
+title: pages.arbitrum
+parent: pages.chains
+grand_parent: pages.tutorials
+nav_order: 4
+---
+
+
+# {%t pages.arbitrum %}
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+Starting with FireFly v1.1, it's easy to connect to public Ethereum chains. This guide will walk you through the steps to create a local FireFly development environment and connect it to the Arbitrum Nitro Goerli Rollup Testnet.
+
+## Previous steps: Install the FireFly CLI
+If you haven't set up the FireFly CLI already, please go back to the Getting Started guide and read the section on how to [Install the FireFly CLI](../../gettingstarted/firefly_cli.md).
+
+[← ① Install the FireFly CLI](../../gettingstarted/firefly_cli.md){: .btn .btn-purple .mb-5}
+
+## Create an `evmconnect.yml` config file
+In order to connect to the Binance Smart Chain testnet, you will need to set a few configuration options for the evmconnect blockchain connector. Create a text file called `evmconnect.yml` with the following contents:
+
+```yml
+confirmations:
+    required: 4
```

Comment:
```suggestion
    required: 4 // choose the number of confirmations you require
```

We should also have a link on each doc to a single place that we have/will-have confirmations detail.
Currently the most relevant place is http://localhost:4000/firefly/overview/public_vs_permissioned.html